src/UI/GameOptionSetupController.java

```diff
@@ -9,6 +14,8 @@
 
 public class GameOptionSetupController implements IController
 {
+  public final int GAME_OPTIONS_CONFIG_KEY = -1; // Player indices start at 0, so -1 should be safe.
```

Comment:
Player indices? This class handles game options, not players, right?

Reply:
Yessir, but it goes in the same file as the player config options. They're numbered from 0, so -1 is safe.